Cesium3DTilesSelection/src/TilesetContentManager.cpp

```diff
@@ -347,6 +395,11 @@ std::vector<CesiumGeospatial::Projection> mapOverlaysToTile(
   return projections;
 }
 
+// Really, these two methods are risky and could easily result in bugs, but
+// they're private methods used only in this class, so it's easier to just tell
+// clang-tidy to ignore them.
```

Comment:
I don't think this is inherently risky, but it has particular semantics that are captured perfectly by the [clang::lifetimebound](https://clang.llvm.org/docs/AttributeReference.html#id11) attribute. Unfortunately that's clang-specific, and clang-tidy doesn't honor it anyway prior to version 20. https://clang.llvm.org/extra/clang-tidy/checks/bugprone/return-const-ref-from-parameter.html
So just disabling the check seems ok for now. It also wouldn't be terrible to change these functions to return a `BoundingVolume` by value.